test/battery/accessibility/odrv_test.toml

```diff
@@ -0,0 +1,24 @@
+# [[source]]
```

Comment:
delete dead code?

Reply:
I actually want to keep that source definition around. It turns out with three sources, I was hitting the A, B, C problem that we currently have with parallel ORC. If/when we migrate ORC to the two-pass method, and are able to generate comprehensive reports, I want to make sure this tests behaves as expected with the three variants.

Reply:
makes sense